.evergreen/package-and-upload-artifact.sh

```diff
@@ -30,8 +21,8 @@ if [ "$(uname)" == Linux ]; then
     -e EVERGREEN_EXPANSIONS_PATH=/tmp/build/tmp/expansions.yaml \
     -e NODE_JS_VERSION \
     -e DISTRIBUTION_BUILD_VARIANT \
+    -e DISTRIBUTION_BUILD_ARCH \
```

Comment:
I don't think we need that anymore... you are essentially inferring the Arch out of the build variant if I understood it correctly 🤔

Reply:
Yes, I added it and forgot to remove it :)